lineapy/db/db.py

```diff
@@ -448,6 +449,11 @@ def get_node_value_path(
         """
         value = self.get_node_value_from_db(node_id, execution_id)
         if not value:
+            track(
+                ExceptionEvent(
+                    ErrorType.DATABASE, "Value not found for the node"
```

Comment:
would clarify here that value "path" was not found. since this is the path to the pickle file now (after MJL's merge, it will just be the filename)